docs/dev_guide/index.rst

```diff
@@ -0,0 +1,23 @@
+===============
+Developer guide
+===============
+
+Installing the latest development version
+-----------------------------------------
+
+BayesianTracker has been tested with Python 3.7+ on OS X, Linux and Win10.
+The tracker and hypothesis engine are mostly written in C++ with a Python wrapper.
+
+If you would rather install the latest development version, and/or
+compile directly from source, you can clone and install from this repo:
+
+.. code:: sh
+
+   $ git clone https://github.com/quantumjot/BayesianTracker.git
+   $ conda env create -f ./BayesianTracker/environment.yml
+   $ conda activate btrack
+   $ cd BayesianTracker
+   $ pip install -e .
```

Comment:
This is a pet hate of mine. If we have `$` in front, then you cannot copy and paste

Reply:
that can be also another future PR when we add the copy-blocks from sphinx and add them but they are not copied.